source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -1593,7 +1593,7 @@ export class TextInput extends Input {
         }
         else {
             this._inputElement = document.createElement("input");
-            this._inputElement.type = "text";
+            this._inputElement.type = (this.style == Enums.InputTextStyle.Email) ? "email" : "text";
```

Comment:
Why not just set type to this.style? Without that most valid styles will simply be ignored.

Reply:
Will do that; I was being selfish only thinking about the email. I will make sure that parse(json) gets a correct default if needed.